pytripgui/app_logic/app_callbacks.py

```diff
@@ -116,6 +116,22 @@ def open_voxelplan_callback(self, patient_item):
         self.global_data.viewcanvases.set_patient(patient)
         return True
 
+    def open_dicom_callback(self, patient_item):
+        dirname = self.parent_gui.browse_folder_path("Open DICOM folder")
```

Comment:
Change variable name to sth like `dir_name`. Now it has potential crash with `os.dirname`